packages/host/tsconfig.json

```diff
@@ -1,3 +1,10 @@
 {
-  "extends": "../../tsconfig.json"
+  "extends": "../../tsconfig.json",
+  "compilerOptions": {
```

Comment:
hm. yes. kbar isn't ready for react 18 yet.. at least as far as typescript is concerned. I've had to temporarily add in some override typings.